lib/easypost/http_client.rb

```diff
@@ -7,18 +7,18 @@ def initialize(base_url, config)
   end
 
   # Execute an HTTP request to the API.
-  def request(method, path, headers = nil, body = nil)
+  def request(method, path, headers = nil, body = nil, api_version = 'v2')
```

Comment:
suggestion: we reference `v2` a couple times. Instead of having them be lose strings, make a constant somewhere and import that so it's the same everywhere and can be easily swapped in the future.

Reply:
added, also created a `Constants` file

Reply:
e-nums! e-nums!

Actually, this is Python...

Con-stants! Con-stants!